src/components/manage/Blocks/Teaser/DefaultBody.jsx

```diff
@@ -60,9 +53,11 @@ const TeaserDefaultTemplate = (props) => {
               {(href.hasPreviewImage || href.image_field || image) && (
                 <div className="image-wrapper">
                   <Image
-                    src={hasImageComponent ? href : defaultImageSrc}
+                    item={image || href}
+                    imageField={image ? image.image_field : href.image_field}
                     alt=""
                     loading="lazy"
+                    responsive={true}
```

Comment:
Ideally, we would set `sizes` here based on the number of columns in the grid containing the teaser.

Reply:
Yeah, I thought about it but isn't the teaser block being reworked? I didn't want to waste time.
Also, the teaser could be used without grid or with a different kind of grid block.
Maybe setting this sizes should be part of a separate work.